src/main/java/core/basesyntax/dao/FruitFileWriter.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.exceptions.NullFileNameException;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FruitFileWriter implements FruitDataWriter {
+    private final String storageOutput;
+    private final String fileName;
```

Comment:
it is better for services not to have state (fields)
it is more flexible to pass this information to method parameters